process/sync/baseSync.go

```diff
@@ -911,7 +920,14 @@ func (boot *baseBootstrap) restoreState(
 
 	err = boot.scheduledTxsExecutionHandler.RollBackToBlock(currHeaderHash)
 	if err != nil {
-		boot.scheduledTxsExecutionHandler.SetScheduledRootHashAndSCRs(currHeader.GetRootHash(), make(map[block.Type][]data.TransactionHandler))
+		gasAndFees := scheduled.GasAndFees{
+			AccumulatedFees: big.NewInt(0),
+			DeveloperFees:   big.NewInt(0),
+			GasProvided:     0,
+			GasPenalized:    0,
+			GasRefunded:     0,
+		}
+		boot.scheduledTxsExecutionHandler.SetScheduledRootHasSCRsAndGas(currHeader.GetRootHash(), make(map[block.Type][]data.TransactionHandler), gasAndFees)
```

Comment:
this is dupplicated code with line 772 from here but also from the baseProcessor RollBackToBlock section and baseBootStrapStorage in the next file in this PR.

Reply:
Here just the gasAndFees param has been added, with zero values.

Reply:
ok